plone/app/event/dx/behaviors.py

```diff
@@ -41,6 +39,23 @@
 from zope.schema.interfaces import IContextAwareDefaultFactory
 
 
+try:
+    # Import fails for Plone < 6.1
+    # version pin of plone.app.z3cform is set to plone.app.z3cform==4.3.2
+    # TextLinesFieldWidget is introduced in plone.app.z3cform==4.4.0
+    from plone.app.z3cform.widgets.text import TextFieldWidget
+except ImportError:
+    from z3c.form.browser.text import TextFieldWidget
+
+try:
+    # Import fails for Plone < 6.1
+    # version pin of plone.app.z3cform is set to plone.app.z3cform==4.3.2
+    # TextLinesFieldWidget is introduced in plone.app.z3cform==4.4.0
+    from z3c.form.browser.textlines import TextLinesFieldWidget
+except ImportError:
+    from z3c.form.browser.textlines import TextLinesFieldWidget
```

Comment:
This is the same import after `ImportError`

Reply:
ah sorry... i updated the import snippet and started the tests